Strip off extra quotes wrapping each link command line

shimmerhuangAMD opened the pull request:
This is to fix #1088

scchan commented on lib/clamp-link.in:
```diff
@@ -290,6 +290,14 @@ do
     IFS=$'\n' read -d '' -r -a LINES < "$REAL_PATH"
     for LINE in ${LINES[@]}
     do
+      #remove the extra colons wrapping the command
```
you mean removing the double-quotes, not colons?

shimmerhuangAMD replied:
Correct, sorry for misleading by wrong word.
Changed the title and updated the commit with correct comments

scchan replied:
Could you update the comment in the code as well? thanks

shimmerhuangAMD replied:
Updated

shimmerhuangAMD committed:
Strip off extra quote wrapping each link command line

Clamp-link reads each link command from response file and pass the link command to linker using following format, which ld does not recognize:

    ld ""-L/aa/bb/cc.o" "/a/b/c.o""

This change is to strip the extra quotes wrapper for each link command line and pass down the command to linker in this way:

    ld "-L/aa/bb/cc.o /a/b/c.o"

shimmerhuangAMD committed:
Update clamp-link.in

Change the comments
